contracts/axelar-gateway/tests/gateway.rs

```diff
@@ -75,12 +78,15 @@ fn validate_message() {
 
     let prev_event_count = env.events().all().len();
 
-    let approved = client.validate_message(
-        &contract_address,
-        &source_chain,
-        &message_id,
-        &source_address,
-        &payload_hash,
+    let approved = invoke_auth!(
```

Comment:
make `assert_invoke_auth_ok` return the unwrapped result instead of creating a new macro

Reply:
having `assert_invoke_auth_ok` return makes it so that all its current uses need to handle the return value (or else a warning for an unused `Result` pops up).  So I think `invoke_auth` serves some purpose for this use case.